spec/rubocop/cop/rspec/change_by_zero_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::ChangeByZero do
+  it 'registers an offense when using `to change(object, :attribute).by(0)`' do
```

Comment:
This docstring says `object`, but the cop only inspects if it's a literal class.